Formula/gojq.rb

```diff
@@ -1,22 +1,18 @@
 class Gojq < Formula
   desc "Pure Go implementation of jq"
   homepage "https://github.com/itchyny/gojq"
-  url "https://github.com/itchyny/gojq/archive/v0.12.0.tar.gz"
-  sha256 "3f280536f53ab6670b89424b0cd295d096b2d2928a5ed5d53bc3a23333051c05"
+  url "https://github.com/itchyny/gojq.git",
+      tag:      "v0.12.0",
+      revision: "203d5e1dfd435460500ea5b00f71eca48bf1cb88"
   license "MIT"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "b8937d8be16fa3f13d0e9f971d2d7e5c0ff0f8cb4cfb02ec53d72344ecd84cf6" => :big_sur
-    sha256 "7ab6960bd90fb64aeff25ce6af2fc96e1bafc2128501e4096e6f4fc442f1034b" => :arm64_big_sur
-    sha256 "522b3e31ccd269471af44917ecde15099000bb78505bbf31d2879faa1af7af4f" => :catalina
-    sha256 "dd0167516da92c42f829858caf8fddbb8ee55f060ee622fd187fc10bd25c0a0d" => :mojave
-  end
+  head "https://github.com/itchyny/gojq.git"
 
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args, "./cmd/gojq"
+    system "make", "build"
+    bin.install "gojq"
```

Comment:
What's wrong with using `go build`?

Reply:
I think we can keep using `go build` here. See the `buildkit` formula for an example of how to set the `ldflags` correctly.

Reply:
I edited, how about now ?

Reply:
Yes, this looks good. I think your commit message should be something like

    gojq: use git checkout for correct version information